VideoLocker/res/layout/fragment_course_dashboard.xml

```diff
@@ -2,13 +2,14 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:background="@color/white"
+    android:layout_height="match_parent"
+    android:background="@color/edx_grayscale_neutral_x_light"
```

Comment:
This should be changed in the window background attribute in the global theme instead of here I guess.